members.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Chakra+Petch:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&family=Mochiy+Pop+One&family=Playpen+Sans:wght@100..800&family=Roboto+Mono:ital,wght@0,100..700;1,100..700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+    <link rel="stylesheet" href="members.css">
+    <title>TGH Members</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo">
+            <img src="images/TechGirlsHubLogo.png " alt="TechGirlsHub Logo" />
+        </div>
+        <nav>
+            <ul>
+                <li><a href="AboutUs/index.html">About Us</a></li>
+                <li><a href="Services.html">Services</a></li>
+                <li><a href="projects.html">Projects</a></li>
+                <li><a href="members.html">Team Members</a></li>
+                <li><a href="contactUs.php">Contact Us</a></li>
+                <li><a href="php/feedbackRatingSubmission.php">Feedback Rating</a></li>

```

Comment:
for the about us , please check if its inside a folder.
the href for feedback rating is feedbackRating.php

Reply:
please resolve if this is done